File: BookLibrary-cdi/backend/src/main/java/com/softserveinc/booklibrary/backend/dto/filtering/AuthorFilter.java
package com.softserveinc.booklibrary.backend.dto.filtering;

import java.math.BigDecimal;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class AuthorFilter {

	private String name;
	private BigDecimal authorRatingFrom;    // todo rename to ratingFrom
	private BigDecimal authorRatingTo;      // todo rename to ratingTo

}

File: BookLibrary-cdi/frontend/src/app/model/author-filter.ts
export class AuthorFilter {
    name: string
    ratingFrom: number
    ratingTo: number

    constructor(name: string, authorRatingFrom: number, authorRatingTo: number) {
        this.name = name;
        this.ratingFrom = authorRatingFrom;
        this.ratingTo = authorRatingTo;
    }
}